src/routes/userRoute.js

```diff
@@ -9,4 +14,8 @@ router.post('/admin-block-user', auth.dev, adminBlockUser);
 router.post('/admin-unblock-user', auth.dev, adminUnblockUser);
 router.post('/auto-follow-users-anon', auth.dev, autoFollowUserAnon);
 
+if (process.env.SET_ALL_FOLLOWER_COUNT === 'true') {
+  router.post('/set-all-follower-count', setAllFollowerCount);
+}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The conditional route addition based on the environment variable `SET_ALL_FOLLOWER_COUNT` is a good practice for feature flagging. However, it's important to ensure that the environment variable is properly set in all environments where this code will run. If not, it could lead to unexpected behavior.
```diff
- if (process.env.SET_ALL_FOLLOWER_COUNT === 'true') {
+ if (process.env.SET_ALL_FOLLOWER_COUNT?.toLowerCase() === 'true') {
    router.post('/set-all-follower-count', setAllFollowerCount);
  }
```
This change ensures that even if the environment variable is set as 'True' or 'TRUE', the condition will still evaluate to true.


<!-- This is an auto-generated comment by OSS CodeRabbit -->